guides/v2.3/graphql/reference/customer.md

```diff
@@ -350,6 +360,267 @@ Attribute |  Data Type | Description
 `attribute_code` | String | Attribute code
 `value` | String | Attribute value
 
+### Customer address region input attributes {#customerAddressRegionInput}
+
+The `customerAddressRegionInput` object can contain the following attributes:
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`region_code` | String | The address region code
+`region` | String | The state or province name
+`region_id` | Int | Uniquely identifies the region
+
+
+### The list of countries codes {#CountryCodeEnum}
```

Comment:
Delete this enumeration. Until the GraphQL reference information is autogenerated, I don't want to document all the enumerations, especially those that are very long, like countries, regions, and currencies. For many of the small enumerations, I list the options in the attribute description.

Reply:
**done**